xmr-btc/src/alice.rs

```diff
@@ -884,10 +884,11 @@ impl State5 {
 
         let s = s_b.scalar + self.s_a.into_ed25519();
 
+        // TODO: Optimized rescan height should be passed for refund as well.
```

Comment:
Can you do this now?

Reply:
I would action it later, because we are focusing on happy-path and I would not optimize for refund yet.